attachment.go

```diff
@@ -19,15 +19,15 @@ func NewAttachment(r *Replication, part *multipart.Part) (*Attachment, error) {
 	// copy headers into Headers
 	contentType := part.Header["Content-Type"][0]
 	contentDisposition := part.Header["Content-Disposition"][0]
-	r.LogTo("Replicate", "attachment contentType: %v", contentType)
-	r.LogTo("Replicate", "attachment contentDisposition: %v", contentDisposition)
+	LogTo("Replicate", "attachment contentType: %v", contentType)
+	LogTo("Replicate", "attachment contentDisposition: %v", contentDisposition)
```

Comment:
I think that including the replication context in the output is valuable - I wouldn't want to just drop it completely.  I'm thinking the approach would be something like https://github.com/couchbaselabs/sg-replicate/pull/28#issuecomment-248752960 - what do you think of that approach?